utils/test/testUtils.js

```diff
@@ -107,6 +107,43 @@ function createMockEllipsoidStackSegmentation({
   }
 }
 
+/**
+ * Adds two concentric circles to each axial slice of the demo segmentation.
+ */
+function createMockEllipsoidVolumeSegmentation({
```

Comment:
Could we import this from a file please?  I like the fact that you have added it here, but I think I'd like to see the imports come from individual files.